elementary/monitor/api/groups/groups.py

```diff
@@ -53,6 +53,12 @@ def _update_dbt_group(
             return
 
         artifact_full_path_split = artifact.normalized_full_path.split(posixpath.sep)
+        if isinstance(artifact, NormalizedExposureSchema):
+            # For exposures, we want the path to be the path in the BI
+            artifact_full_path_split = artifact_full_path_split[
+                :2
+            ] + artifact.fqn.split("/")
```

Comment:
first two parts of the full path are the package name and "exposures", which we want to keep

Reply:
can we add a test?

Reply:
I'll actually replace with more straightforward logic